docs/core/tutorials/top-level-templates.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Template changes in .NET 6"
+description: The console app template now uses top-level statements. Understand what's changed and how to use existing learning materials with the new program.
+ms.date: 07/13/2021
+---
+# New templates generate top-level statements
+
+Starting with the .NET 6 Preview 7 SDK, the console app template generates the following code:
+
+```csharp
+Console.WriteLine("Hello, World!");
+// See https://aka.ms/new-console-template for more information
+```
+
+The new output uses two recent C# features that simplify the code you need to write for a program.  Traditionally, the console template generated the following code:
+
+```csharp
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace MyApp // Note: actual namespace depends on the project name.
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
+        }
+    }
+}
+```
+
+These two forms represent the same program. Both are valid with C# 10.0.  When you use the newer version, you need to write the body of the `Main` method. You don't need to write the other program elements by hand. You have two options you can follow with existing tutorials:
+
+- Use the new program style, adding new top-level statements as you add features
```

Comment:
I don't think "top-level statements" will become a way to describe these statements. I suspect they will just be statements. I'm not sure this is quite right. There may be a third way.:

```suggestion
- Use the new program style, just including the code that makes up your application
```